### ISSUE TITLE
Feature request: Excess Noise in WFS

### ISSUE THREAD

**Issue by jcranney** [ANR-COMPASS/shesha#31]
## Description

In other AO simulation tools (e.g., YAO, PASSATA) it is possible to simulate excess noise in the WFSs (in addition to the standard RON, photon noise, etc.). An example implementation can be found in the [yao_fast.c](https://github.com/frigaut/yao/blob/master/yao_fast.c#L1331) file, line 1331. This should be relatively straightforward to implement and would improve the practicality of COMPASS.

In short, for a given `excessnoise (default=1.0f)`, the flux is scaled by `1/(excessnoise*excessnoise)` before computing the Poisson distributed pixel intensities, then afterwards the intensities are rescaled by `excessnoise*excessnoise` to restore the correct flux before the readout noise, etc, is added.
